maven-enforcer-plugin/src/main/java/org/apache/maven/plugins/enforcer/EnforceMojo.java

```diff
@@ -115,6 +120,14 @@ public class EnforceMojo extends AbstractMojo {
     @Parameter(required = false, property = "rules")
     private String[] commandLineRules;
 
+    /**
+     * Array of Strings that matches the EnforcerRules to skip.
+     *
+     * @since 3.2.0
+     */
+    @Parameter(required = false, property = "enforcer.skipRules")
+    private String[] commandLineRulesToSkip;
```

Comment:
I don't really like the name here, since this is just a property value, which may not be supplied via command line at all, but instead via a property in pom.xml. I understand that the main use case is likely to override the rules via cmd line, but still seems a bit weird.

On the other hand, since the other property (`commandLineRules`) is already using this name it only makes sense to be consistent. So either rename both or none (hopefully renaming the field would not be considered a breaking change since it is `private`).

Reply:
field name determinate configuration item for plugin, but can be changed by `@Parameter(name=...)`

Reply:
Thanks for the additional info. I guess these are a bit specific since users likely won't use them directly in pom.xml config section, but rather via a property.

One more thing I am not sure about is the property name. `skipRules` may suggest that it is a boolean property, which is not the case here. I could not come up with a better name
 * `rulesToSkip` seems weird as well
 * `skippedRules` - not sure about the past tense?

Reply:
`skipRules` and `rulesToSkip` - are both ok for me

Reply:
I used the `skipRules` in the end, since it seems to better match the `rules` one to include rules. And it is also shorter / easier to write by hand, which I think is a plus since it is supposed to be used from cmd line.